test/integration/multiple.done.js

```diff
@@ -4,25 +4,95 @@ var args   = [];
 
 describe('multiple calls to done()', function() {
   var res;
-
   this.timeout(1000);
 
-  before(function(done) {
-    run('multiple.done.js', args, function(err, result) {
-      res = result;
-      done(err);
+  describe('from a spec', function() {
+    before(function(done) {
+      run('multiple.done.js', args, function(err, result) {
+        res = result;
+        done(err);
+      });
+    });
+
+    it('results in failures', function() {
+      assert.equal(res.stats.pending, 0);
+      assert.equal(res.stats.passes, 1);
+      assert.equal(res.stats.failures, 1);
+      assert.equal(res.code, 1);
+    });
+
+    it('throws a descriptive error', function() {
+      assert.equal(res.failures[0].err.message,
+        'done() called multiple times');
+    });
+  });
+
+  describe('with multiple specs', function() {
+    before(function(done) {
+      run('multiple.done.specs.js', args, function(err, result) {
+        res = result;
+        done(err);
+      });
+    });
+
+    it('results in a failure', function() {
+      assert.equal(res.stats.pending, 0);
+      assert.equal(res.stats.passes, 2);
+      assert.equal(res.stats.failures, 1);
+      assert.equal(res.code, 1);
+    });
+
+    it('correctly attributes the error', function() {
+      assert.equal(res.failures[0].fullTitle, 'suite test1');
+      assert.equal(res.failures[0].err.message,
+        'done() called multiple times');
     });
   });
 
-  it('results in failures', function() {
-    assert.equal(res.stats.pending, 0);
-    assert.equal(res.stats.passes, 1);
-    assert.equal(res.stats.failures, 1);
-    assert.equal(res.code, 1);
+  describe('from a before hook', function() {
+    before(function(done) {
+      run('multiple.done.before.js', args, function(err, result) {
+        res = result;
+        done(err);
+      });
+    });
+
+    it('results in a failure', function() {
+      assert.equal(res.stats.pending, 0);
+      assert.equal(res.stats.passes, 1);
```

Comment:
> Why is passes 1?

You're using the wrong bin. Use the project's bin, rather than global mocha.
``` javascript
danielstjules:~/git/mocha (issue-1798)
$ ./bin/mocha test/integration/fixtures/multiple.done.before.js --reporter JSON
{
  "stats": {
    "suites": 1,
    "tests": 1,
    "passes": 1,
    "pending": 0,
    "failures": 1,
    "start": "2015-09-14T16:28:29.589Z",
    "end": "2015-09-14T16:28:29.658Z",
    "duration": 69
  },
```